subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -33,6 +33,12 @@ abstract class InstrumentationTestsTask @Inject constructor(
     private val workerExecutor: WorkerExecutor
 ) : DefaultTask() {
 
+    init {
+        outputs.upToDateWhen {
+            false
+        }
+    }
```

Comment:
?

Reply:
Accidentally left. I'll remove it. It was necessary on the early steps of this task.

Reply:
Fixed